README.md

```diff
@@ -523,11 +523,11 @@ E.g.:
 Request
 
 ```
-curl -L https://gratipay.com/username/payment-instructions.json \
```

Comment:
Good idea. Should also change `/%username/public.json` to `/~username/public.json` for uniformity.